sklearn/preprocessing/data.py

```diff
@@ -1917,3 +1920,38 @@ def transform(self, X):
         """
         return _transform_selected(X, self._transform,
                                    self.categorical_features, copy=True)
+
+
+def _boxcox(X):
```

Comment:
But I think you could pass the `stats.boxcox` as such and extract the desired output in one go? Would be cleaner? No strong opinion though.